extensions/wikia/CategoryExhibition/templates/form.tmpl.php

```diff
@@ -1,40 +1,25 @@
 <div class="category-gallery-form">
-	<? if ( $displayType == 'exhibition' ): ?>
-		<?= wfMessage( 'category-exhibition-sorttype' )->escaped(); ?>
-		<?
-		$dropdown = array();
-		foreach ( $sortTypes as $sortType ) {
-			$el = array();
-			if ( $current == $sortType ) {
-				$el["class"] = "selected";
-			}
-			$el["href"] = "$path?sort=$sortType&display=$displayType";
-			$el["id"] = Sanitizer::escapeId( "category-exhibition-form-$sortType" );
-			$el["text"] = wfMessage( "category-exhibition-$sortType" )->escaped();
-			$dropdown[] = $el;
+	<?= wfMessage( 'category-exhibition-sorttype' )->escaped(); ?>
+	<?php
+	$dropdown = array();
```

Comment:
just a note: some of the code in this PR uses the `[]` syntax while there are still some places that use `array()`